### PR TITLE
platforms/azure: Remove etcd-lb and connect to etcd directly

### LINKED ISSUE #464: azure: etcd machines are on the Internet and the VMs have a static password
If I'm reading the Terraform correctly, this is putting etcd on the Internet. Is this intentional? Why public over an internal LB for etcd?

https://github.com/coreos/tectonic-installer/blob/master/modules/azure/etcd/lb.tf#L14

Also, is it intentional that the admin password is hardcoded? Is that just for ease of access since SSH isn't exposed to the internet for those machines?

https://github.com/coreos/tectonic-installer/blob/9bdb83ee488c798ee1760dd3ff3854fd3be86179/modules/azure/etcd/etcd.tf#L33

### PR DESCRIPTION
**Pull request by discordianfish**
This closes #464.

There is no reason for using an lb for that, so it should be removed to reduce attack surface and simplify configuration.

### CODE REVIEW

File: platforms/azure/tectonic.tf [hunk 1]
```diff
@@ -23,7 +23,7 @@ module "bootkube" {
   oidc_groups_claim   = "groups"
   oidc_client_id      = "tectonic-kubectl"
 
-  etcd_endpoints   = ["${module.vnet.etcd_public_ip}"]
```
Comment (discordianfish) [inline platforms/azure/tectonic.tf]: Should we use the DNS name generated by `modules/azure/dns/etcd.tf`? That way we don't need to update the masters if the etcd ips change. Not sure what the reasoning was to use IPs here.

**Comment by justaugustus** [comment]
With the move of LB code into `modules/vnet`, the etcd-lb was removed.
I believe this can be closed out.

**Comment by alexsomesan** [comment]
@justaugustus Right, this has been solved as you mentioned.

**Pull request closed without merging**